docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "External WebLogic Clients"
+date: 2019-11-21T21:23:03Z
+draft: false
+weight: 6
+---
+
+### Approaches
+
+There are two supported approaches for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster: [Load Balancer Tunneling](#load-balancer-tunneling) and [Kubernetes Node Ports](#kubernetes-node-ports).
+
+> __NOTE:__ This FAQ is for remote EJB and JMS clients - not JTA clients. The operator does not currently support external WebLogic JTA access to a WebLogic cluster, as (a) external JTA access requires each server in the cluster to be individually addressable by the client, but this conflicts with (b) the current operator requirement that a network channel in a cluster have the same port across all servers in the cluster.
```

Comment:
For the Hugo UI, we use this code to surround NOTES text:
{{% notice note %}}
{{% /notice %}}

Also, I would suggest that you use the designation (a.) and (b.), because in the first instance, it looks like you are stating as (a) external... (instead of as an external ...,) but you are denoting neither. 
Or, as you've done in the Config Overrides doc, use the capital letters, A and B.
(Please apply this change globally,)